ttg/ttg/serialization/boost.h

```diff
@@ -0,0 +1,35 @@
+//
+// Created by Eduard Valeyev on 5/3/21.
```

Comment:
Should we use proper copyright headers? (might be a question beyond the files added here)

Reply:
yes, should discuss on Friday